rust/agama-lib/src/error.rs

```diff
@@ -33,6 +35,8 @@ pub enum ServiceError {
     UnsuccessfulAction(String),
     #[error("Unknown installation phase: '{0}")]
     UnknownInstallationPhase(u32),
+    #[error("Backend call failed with status '{0}' and text '{1}'")]
```

Comment:
NP, status is numeric
```suggestion
    #[error("Backend call failed with status {0} and text '{1}'")]
```